geoportal/c2cgeoportal_geoportal/templates/dynamic.js

```diff
@@ -67,12 +67,21 @@ wfs_permalink.update(interface_config.get('wfs_permalink', {}))
     else:
         query = {}
     if 'themes' in request.matchdict:
+        no_redirect_url = request.route_url(
+            interface_config['redirect_interface'] + 'theme',
+            themes=request.matchdict['themes'],
+            _query=no_redirect_query
+        )
         url = request.route_url(
             interface_config['redirect_interface'] + 'theme',
             themes=request.matchdict['themes'],
             _query=no_redirect_query
         )
     else:
+        no_redirect_url = request.route_url(
+            interface_config['redirect_interface'],
+            _query=no_redirect_query
+        )
         url = request.route_url(
             interface_config['redirect_interface'],
             _query=no_redirect_query
```

Comment:
What is the difference between `no_redirect_url` and `url` ?
Maybe I miss something, but I do not see it.
As I see, at this point : `no_redirect_url === url`

Reply:
no_redirect_url shoudb have the param `no_redirect=t`... => my PR is incorrect :-(

Reply:
See  #4492